parser/output/raylib_api.lua

```diff
@@ -335,6 +335,12 @@ return {
       type = "UNKNOWN",
       value = "SHADER_LOC_MAP_METALNESS",
       description = ""
+    },
+    {
+      name = "GetMouseRay",
```

Comment:
raylib parser output shouldn't be modified on this PR.

Reply:
This is modified by the CI bot automatically. I have reverted it.